### ISSUE TITLE
Resolution options not screen reader accessible

### ISSUE THREAD

**Issue by danirabbit** [elementary/switchboard-plug-display#452]
### What Happened?

When navigating the configure display popover, resolution options are read by the screen reader as "panel"

### Steps to Reproduce

1. Turn on the screen reader
2. Open the Configure Display popover for a display
3. Navigate to the Resolutions combobox and open it via keyboard
4. Notice no resolutions are read out, only "panel"

### Expected Behavior

This needs to be ported to Gtk.DropDown which might just fix things, but it should be screen reader accessible

### OS Version

8.x (Circe)

### OS Architecture

amd64 (on most hardwares)

### Session Type

Secure Session (Wayland, This is the default)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell

```
